pom.xml

```diff
@@ -80,6 +80,7 @@
         <maven.compiler.release>11</maven.compiler.release>
 
         <repoServerHost>oss.sonatype.org</repoServerHost>
```

Comment:
In the DSL project, this is `s01.oss.sonatype.org`. Is that expected to be different?